docs/roles/system_manager/manage_deployments.md

```diff
@@ -74,6 +74,18 @@ This is because only Nvidia GPUs support the drivers and CUDA libraries installe
 
 To test that a GPU enabled VM is working as expected, log into the SRE and type `nvidia-smi` into the terminal.
 
+## {{crown}} Performing operations that require System Administrator privileges
+
+If you need to perform any operations in the SRE that require root access, you will need to log into the `compute` VM via the Serial Console in the Azure Portal.
+
+```{include} snippets/01_console.partial.md
+:relative-images:
+```
+
+```{warning}
+Tier 0 and 1 SREs have internet access and as such, installing anything as root that involves downloading from the internet could introduce malicious software being introduced to the SRE.
```

Comment:
What are you trying to warn about here? You could fetch and run malicious software from the internet as a normal user too.

Reply:
@JimMadge you're right, I think what I really wanted the warning to convey was "be careful when installing things as root" - I think I'll just remove this though, because I think "System Managers" should already know to be careful